helm/dendrite/README.md

```diff
@@ -72,12 +66,12 @@ Create a folder `appservices` and place your configurations in there.  The confi
 | configuration.metrics.basic_auth.password | string | `"metrics"` | HTTP basic authentication password |
 | configuration.metrics.basic_auth.user | string | `"metrics"` | HTTP basic authentication username |
 | configuration.metrics.enabled | bool | `false` | Whether or not Prometheus metrics are enabled. |
-| configuration.mscs | list | `[]` | Configuration for experimental MSC's. (Valid values are: msc2836 and msc2946) |
-| configuration.profiling.enabled | bool | `false` | Enable pprof |
+| configuration.mscs | list | `["msc2946"]` | Configuration for experimental MSC's. (Valid values are: msc2836 and msc2946) |
```

Comment:
I don't understand why we are inconsistent on key names. This is `configuration.mscs` as if `configuration` refers to the config YAML file, but then we have `clientapi.registration.registration_disabled` and others which lack the `configuration` prefix :S also `config` would be slightly shorter/nicer to type.

Reply:
Updated, should be more consistent now.
The sections are based on the Dendrite config.